src/parser/smt2/smt2.cpp

```diff
@@ -51,7 +51,10 @@ void Smt2::addArithmeticOperators() {
   Parser::addOperator(kind::LEQ);
   Parser::addOperator(kind::GT);
   Parser::addOperator(kind::GEQ);
-  
+}
```

Comment:
I think POW is actually a non-transcendental function (it was there before we supported transcendental functions).  It's not standard.  However, it probably makes sense to have this as a non-transcendental function.